easybuild/easyconfigs/i/intel/intel-2016.02.eb

```diff
@@ -0,0 +1,25 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+easyblock = "Toolchain"
+
+name = 'intel'
+version = '2016.02'
```

Comment:
yeap, thanks for the GCC-4.x PR.

IMHO, `binutils/2.26` would likely be preferable by EB community, 
in the name of supporting more architectures;
this might, just might, make it less of an issue the 5.1 vs 5.3 debate... (fingers crossed)